maat-court-data-api/src/test/java/gov/uk/courtdata/dces/service/FdcContributionsServiceTest.java

```diff
@@ -197,6 +199,90 @@ void testDrcUpdateSuccessHasErrorText(){
         assertEquals(currDate.getYear(), errorEntity.getDateCreated().getYear());
     }
 
+    @Test
+    void testCreateFdcContribution() {
+        ArgumentCaptor<FdcContributionsEntity> captor = ArgumentCaptor.forClass(FdcContributionsEntity.class);
+
+        CreateFdcContributionRequest request = CreateFdcContributionRequest.builder()
+                .repId(11212)
+                .lgfsComplete("N")
+                .agfsComplete("Y")
+                .status(REQUESTED)
+                .build();
+
+        FdcContributionsEntity entity = FdcContributionsEntity.builder()
+                .id(1)
+                .repOrderEntity(RepOrderEntity.builder().id(1).build())
+                .lgfsComplete(request.getLgfsComplete())
+                .agfsComplete(request.getAgfsComplete())
+                .status(request.getStatus())
+                .build();
+
+        when(fdcContributionsRepository.save(any(FdcContributionsEntity.class))).thenReturn(entity);
+
+        Integer result = fdcContributionsService.createFdcContribution(request);
+
+        verify(fdcContributionsRepository).save(captor.capture());
+        FdcContributionsEntity capturedEntity = captor.getValue();
+
+        assertNotNull(result);
```

Comment:
For multiple assertions can you use `assertAll`?
see [QueueMessageMaintenanceSchedulerTest.java](https://github.com/ministryofjustice/laa-maat-court-data-api/blob/92cb7eb619523d3330be1f76f8e4320c5dd293a6/maat-court-data-api/src/test/java/gov/uk/courtdata/integration/job/QueueMessageMaintenanceSchedulerTest.java) for an example.

Reply:
OK I wasn't aware of that, np